examples/ASDKgram/Sample/AppDelegate.m

```diff
@@ -0,0 +1,116 @@
+  tabBarController.viewControllers        = @[uikitHomeFeedNavCtrl, asdkHomeFeedNavCtrl];
+  tabBarController.selectedViewController = asdkHomeFeedNavCtrl;
+  tabBarController.delegate               = self;
+  
+  // Nav Bar appearance
+  [[UINavigationBar appearance] setBarTintColor:[UIColor darkBlueColor]];
+  [[UINavigationBar appearance] setTranslucent:NO];
+  NSDictionary *attributes = @{NSForegroundColorAttributeName:[UIColor whiteColor]};
+  [[UINavigationBar appearance] setTitleTextAttributes:attributes];
+  // make the status bar have white text (changes after scrolling, but not on initial app startup)
+  // UINavigationController does not forward on preferredStatusBarStyle calls to its child view controllers.
+  // Instead it manages its own state...http://stackoverflow.com/questions/19022210/preferredstatusbarstyle-isnt-called/19513714#19513714
+  uikitHomeFeedNavCtrl.navigationBar.barStyle = UIBarStyleBlack;
+  asdkHomeFeedNavCtrl.navigationBar.barStyle  = UIBarStyleBlack;
+  
+  self.window.rootViewController = tabBarController;
+  [self.window makeKeyAndVisible];
+  
+  // opaque status bar background
+  self.statusBarOpaqueUnderlayView                 = [[UIView alloc] init];
+  self.statusBarOpaqueUnderlayView.backgroundColor = [UIColor darkBlueColor];
+  self.statusBarOpaqueUnderlayView.frame           = [[UIApplication sharedApplication] statusBarFrame];
+  if ( ![[UIApplication sharedApplication] isStatusBarHidden] ) {
+    [[[UIApplication sharedApplication] keyWindow] addSubview:self.statusBarOpaqueUnderlayView];
+  }
+  [[NSNotificationCenter defaultCenter] addObserver:self
+                                           selector:@selector(handleDidChangeStatusBarOrientationNotification:)
+                                               name:UIApplicationWillChangeStatusBarOrientationNotification
+                                             object:nil];
+  return YES;
+}
+
+// UITabBarControllerDelegate
+- (void)tabBarController:(UITabBarController *)tabBarController didSelectViewController:(UIViewController *)viewController;
+{
+  if ([viewController isKindOfClass:[UINavigationController class]]) { 
+    NSArray *viewControllers = [(UINavigationController *)viewController viewControllers];
+    UIViewController *rootViewController = viewControllers[0];
+    if ([rootViewController conformsToProtocol:@protocol(PhotoFeedViewControllerProtocol)]) {
+      // FIXME: the dataModel does not currently handle clearing data during loading properly
+//      [(id <PhotoFeedViewControllerProtocol>)rootViewController resetAllData];
+    }
+  }
+}
+
+- (void)handleDidChangeStatusBarOrientationNotification:(NSNotification *)notification;
+{
+  NSNumber *newOrientation = [notification.userInfo objectForKey: UIApplicationStatusBarOrientationUserInfoKey];
+  NSInteger newOrientationInt = [newOrientation integerValue];
+  if (UIInterfaceOrientationIsLandscape(newOrientationInt)) {
+    [self.statusBarOpaqueUnderlayView removeFromSuperview];
```

Comment:
Why is it necessary to remove this ever?  Status bar should show in landscape too.